Xamarin.Essentials/Connectivity/Connectivity.uwp.cs

```diff
@@ -15,6 +16,18 @@ public static partial class Connectivity
         static void StopListeners() =>
              NetworkInformation.NetworkStatusChanged -= NetworkStatusChanged;
 
+        static SignalStrength PlatformSignalStrength()
+        {
+            var signalStrength = NetworkInformation.GetConnectionProfiles()
+                .Where(profile => profile != null && profile.IsWlanConnectionProfile)
+                .Select(profile => profile.GetSignalBars())
+                .Where(signalBar => signalBar.HasValue)
+                .OrderBy(strength => strength).FirstOrDefault();
```

Comment:
Total styling preference here, but the fact that each line is a separate method, except for the FirstOrDefault is freaking me out a bit :) OCD guy here.

Reply:
Understandable, changed it :)